api/metadata.proto

```diff
@@ -78,12 +78,16 @@ message RxMetadata {
   bytes uplink_token = 15;
   // Index of the gateway channel that received the message.
   uint32 channel_index = 17 [(validate.rules).uint32 = {lte: 255}];
+  // LR-FHSS hopping width (grid), in kHz.
+  string lrfhss_hopping_width = 19;
+  // Frequency drift in Hz between start and end of an LR-FHSS packet (signed).
+  int32 lrfhss_frequency_drift = 20;
```

Comment:
Either put these in a new `LRFHSS` sub message, or remove `lrfhss_` and see if they will be reused for other modulations. The latter is what we've been doing so far here.